src/Altinn.Platform/Altinn.Platform.Events/Events/Services/CosmosService.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Net;
+using System.Threading.Tasks;
+using Altinn.Platform.Events.Configuration;
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.Repository;
+using Altinn.Platform.Events.Services.Interfaces;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Events.Services
+{
+    /// <inheritdoc/>
```

Comment:
I don't like using inherit doc on the class. An interface and a class are two very different things.

Reply:
changed to a summary